uaclient/status.py

```diff
@@ -507,6 +507,22 @@ def __init__(
 applicable law.
 """
 
+MESSAGE_APT_PROXY_CONFIG_HEADER = """\
+/*
+ * Autogenerated by ubuntu-advantage-tools
+ * Do not edit this file directly
+ *
+ * To change what ubuntu-advantage-tools sets here, edit the apt_http_proxy
+ * and apt_https_proxy fields in /etc/ubuntu-advantage/uaclient.conf. Then
+ * run:
+ *
+ * ua refresh config
```

Comment:
Since we are documenting ua refresh cmd TBD, I think we should also document this directly in the header of the uaclient.conf we ship
Please also add something comparable mentioning `ua refresh config` to https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient.conf

Reply:
yeah on second thought it feels weird including a reference to the non-existent-yet command in this PR. Since I preemptively started the `ua refresh [config|contract]` pr over at #1678 I think I'll remove this reference to it and add this comment as well as a comment in uaclient.conf to #1678

Reply:
+1 on that suggestion.